SixTrack/scatter.s90

```diff
@@ -435,6 +436,13 @@ subroutine scatter_parseGenerator(getfields_fields,getfields_lfields,getfields_n
         ! PPBEAMELASTIC b2
         scatter_fdata(tmpIdx+4) = round_near(errno,getfields_lfields(8)+1,getfields_fields(8))
         if(errno .ne. 0) call rounderr(errno,getfields_fields,8,scatter_fdata(tmpIdx+4))
+
+        if(getfields_nfields .eq. 9) then
+            ! PPBEAMELASTIC corss-section
```

Comment:
Spelling

Reply:
It's a typo in comment. It makes no difference whatsoever. Just fix it when you see it.